### PR TITLE
Update README.txt

### PR DESCRIPTION
**Pull request by Source-X**
Added some truth in it. I lied, but now I am not lying. Partially.

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -1,4 +1,8 @@
 // THIS IS TEST! THIS IS TEST! THIS IS TEST! THIS IS TEST! THIS IS TEST!
 // Cake is a lie.
 
-// All random shit goes here.
```
Comment (Source-X) [inline README.txt:4]: Huh? why the "-" at the beginning? This line was not removed.

File: README.txt [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
\ No newline at end of file
+// All random shit goes here.
```
Comment (Source-X) [inline README.txt:4]: and why is this line repeated to make it look like it's added?

File: README.txt [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+
+
+//But the truth is... cake is not a lie. DID YOU NOT SEE THE ENDING?!
+//For branch testing purpose... or is it another lie?
```
Comment (Source-X) [inline README.txt:8]: How can I trust a liar?

CHANGES APPROVED
-Jensen

File: README.txt (hunk 1 above)
Comment (vJXSs) [inline README.txt:4]: The '-' shows it is removed.

**Pull request merged**